### PR TITLE
Add MLIR python bindings to the premerge bot config

### PR DESCRIPTION
**Pull request by joker-eph**
This matches what builkite is doing right now.

Also add myself to the notification for this bot.

### CODE REVIEW

File: buildbot/osuosl/master/config/status.py [hunk 1]
```diff
@@ -407,14 +407,14 @@ def getReporters():
...
-            extraRecipients = ["llvm-premerge-buildbots@google.com"],
+            extraRecipients = ["llvm-premerge-buildbots@google.com", "joker.eph@gmail.com"],
             mode = "failing",
             builders = ["premerge-monolithic-windows", "premerge-monolithic-linux"]),
         reporters.MailNotifier(
             fromaddr = "llvm.buildmaster@lab.llvm.org",
             sendToInterestedUsers = False,
             messageFormatter = LLVMInformativeMailNotifier,
-            extraRecipients = ["szakharin@nvidia.com"],
+            extraRecipients = ["szakharin@nvidia.com""],
```
Comment (gkistanova) [inline buildbot/osuosl/master/config/status.py:417]: Double quotation marks at the end of the line looks like a mistype. Did you mean adding yourself to the list of extra recipients?

**Review by gkistanova — COMMENTED** [review]
Thanks, Mehdi. The patch looks fine with one issue.

File: buildbot/osuosl/master/config/status.py (hunk 1 above)
Comment (joker-eph) [reply buildbot/osuosl/master/config/status.py]: I added myself here and realized it was the wrong bot and didn't undo it properly. Thanks for catching!

I fixed it now.

**Comment by joker-eph** [comment]
@lnihlen are you the owner of this bot right now?

**Comment by lnihlen** [comment]
> @lnihlen are you the owner of this bot right now?

Not to my knowledge. @metaflow might know better, but as far as I know all the Google presubmit infra lives over at https://github.com/google/llvm-premerge-checks/.

**Comment by metaflow** [comment]
LGTM (I assume this flag does not require any additaion VM changes).

**Review by metaflow — APPROVED** [review]

**Comment by joker-eph** [comment]
I assume this is the same VM configuration as the premerge?

**Comment by joker-eph** [comment]
> Not to my knowledge. @metaflow might know better, but as far as I know all the Google presubmit infra lives over at https://github.com/google/llvm-premerge-checks/.

@lnihlen I don't know if you sync with @metaflow offline, but this buildbot is running post-merge and supposed to cover the same config as the premerge. So I assume this is handle on the same group of VM somehow.

This allows to keep track of the health of the pre-merge config at HEAD of the main branch: https://lab.llvm.org/buildbot/#/builders/271

**Comment by joker-eph** [comment]
@lnihlen @metaflow this broke the buildbot apparently, seems like the image isn't the same as the pre merge?
The failure is caused because python is older and/or missing numpy:

```
CMake Error at C:/BuildTools/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Python3 (missing: Python3_NumPy_INCLUDE_DIRS NumPy) (found
  suitable version "3.9.7", minimum required is "3.6")
```

**Pull request merged**